Sming/Libraries/ThingSpeak/README.md

```diff
@@ -0,0 +1,61 @@
+# Sming - ThingSpeak driver libary
+## Usage
```

Comment:
Before even starting to talk about usage you can add two or three sentences explaining what is ThingSpeak. You can quote the information from their website (https://mathworks.com/help/thingspeak/getting-started-with-thingspeak.html) "ThingSpeak is an IoT platform that enables you to collect, store, analyze, visualize, and act on data from sensors or actuators, such as Arduino®, Raspberry Pi™, BeagleBone Black, and other hardware. For example, with ThingSpeak you can create sensor-logging applications, location-tracking applications, and a social network of things with status updates, so that you could have your home thermostat control itself based on your current location."